Azure/azure-cli
# [Webapp] Bugfix: az webapp vnet-integration add - fix error handling if bad vnet name, support vnet resource ID

**Description**  
closes #13863, #11454 , #12876

Currently, `az webapp vnet-integration add` command gives a non descriptive error when an invalid Vnet name is passed using --vnet. The error shown is: 'The api-version 2020-04-01 is in valid...' 
Also, --vnet parameter should support using both vnet name and vnet resource ID. Currently, it only supports using vnet name

**Testing Guide**  
- `az webapp vnet-integration add -g RG -n WebappName --vnet=nonExistentVnet --subnet=default` should return error stating that vnet is not found. Currently, it fails with error saying 'The api-version 2020-04-01 is in valid...'
- `az webapp vnet-integration add -g RG -n WebappName --vnet="/subscriptions/pathToVnetResourceId" --subnet=default` should work. Currently, command does not support using Vnet resource ID

---

This checklist is used to make sure that common guidelines for a pull request are followed.
